src/NServiceBus.Core.Tests/Encryption/ValidationFixture.cs

```diff
@@ -0,0 +1,166 @@
+namespace NServiceBus.Core.Tests.Encryption
+{
+    using System.Configuration;
+    using NServiceBus.Config;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class ValidationFixture
+    {
```

Comment:
Nitpick: We don't have an agreed naming convention but the informal standard seems to be `Tests` instead of `Fixture`